.github/main.workflow

```diff
@@ -1,30 +0,0 @@
-workflow "Linters" {
-  on = "push"
-  resolves = ["Black Linter", "Flake8 Linter"]
-}
-
-action "Build" {
-  uses = "jefftriplett/python-actions@ba888adab5b57956dce82b7dfc06e87b65090a6d"
-  args = "pip install flake8 black"
-  env = {
-    VENV_DIR = "/opt/"
-  }
-}
-
-action "Black Linter" {
```

Comment:
I'm curious why this is removed, is this github action not good enough?

Reply:
In another PR I said the following:
>After tests it doesn't work properly. GitHub is building on the latest base branch commit instead of the latest fork's branch commit.

So what happened is that GH Action does build on push/pull_request events but when it's coming from forks, it will build the base branch (master of the mirumee repo).

For now, I did not find any solution for that, so we will be using tox instead.

Reply:
Thanks for reply. It's working for PR but PR is from forks right? What do you mean by ```coming from forks```.

Reply:
PR can also come from Mirumee members that create a PR through a branch in `mirumee/saleor`, those would get checked by the GH-Action. But not the people that create a PR with a branch coming from another repository.

Reply:
That's sad. That makes gh-actions quite unusable. 😢

Reply:
There should be a way! Just didn't have time to dig it :) I am working on other actions and waiting from the GitHub support to give me access to it. So I will see from them.